src/swtpm/swtpm.c

```diff
@@ -96,6 +96,7 @@ static void usage(FILE *file, const char *prgname, const char *iface)
     "\n"
     "-p|--port <port> : use the given port\n"
     "-f|--fd <fd>     : use the given socket file descriptor\n"
+    "-F|--ctrlfd <fd> : use the given socket file descriptor for control channel\n"
     "-t|--terminate   : terminate the TPM once a connection has been lost\n"
     "-d|--daemon      : daemonize the TPM\n"
     "--ctrl type=[unixio|tcp][,path=<path>][,port=<port>[,bindaddr=address[,ifname=ifname]]][,fd=<filedescriptor]\n"
```

Comment:
My initial implementation was used '--ctrl' later i felt it would be better to match  -f , as it  is ment fir client fd :). Anyways i will do the change and re-submit.